templates/taxi/driver_detail.html

```diff
@@ -4,7 +4,9 @@
   <h1>
     Username: {{ driver.username }}
   </h1>
-
+  <p><b><a href="{% url 'taxi:driver-delete' driver.id %}">Click here to delete this driver</a></b></p>
```

Comment:
According to the checklist, you should add a `Cancel` button apart from `Delete` one. The `Cancel` button will lead to the previous page the user was on.